.github/workflows/update.yml

```diff
@@ -13,19 +13,18 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-        with:
-          token: ${{ secrets.BOT_GITHUB_PAT }}
       - run: cd grouparoo && npm install
       - run: sudo npm install -g npm-check-updates
       - run: cd grouparoo && npm-check-updates -u
```

Comment:
Maybe scope this one to only `@grouparoo/*` pacakages too?

Reply:
wasn't working with `@grouparoo/*` because we also have the `grouparoo` package here. Switching to using `/grouparoo/` (it can take in a regex)